packages/element/src/create-interpolate-element.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * Internal dependencies
+ */
+import { createElement, Fragment, isValidElement } from './react';
```

Comment:
Should we import from `..` itself? It's a nit pick, not a big deal, I'll leave it up to you. :)

Reply:
wouldn't that create potential for circular imports?

Reply:
The way it's code now it's fine. We use the same approach nearly everywhere.

> wouldn't that create potential for circular imports?

I saw some circular imports and they are properly resolved, but I agree that we should avoid them.

Reply:
Yeah, you're right. :)